rpcs3/Emu/System.cpp

```diff
@@ -512,20 +532,20 @@ bool Emulator::BootGame(const std::string& path, const std::string& title_id, bo
 	if (direct && fs::exists(path))
 	{
 		m_path = path;
-		Load(title_id, add_only, force_global_config);
-		return true;
+		return Load(title_id, add_only, force_global_config);
 	}
 
-	bool success = false;
+	bool found_any = false;
+	game_boot_result last_status = generic_error;
```

Comment:
found_any is unnecessary.
Set the initial error to nothing_to_boot.
```suggestion
	game_boot_result result = game_boot_result::nothing_to_boot;
```